src/styles/blocks/main.scss

```diff
@@ -0,0 +1,26 @@
+.container {
+  padding: 50px 40px;
+
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
```

Comment:
The value for 'grid-template-columns' here can be simplified. Since you are repeating the same size column one time, you can use '200px' directly without the 'repeat' function.